llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -1420,7 +1425,7 @@ class TargetTransformInfo {
       unsigned Opcode, Type *InputTypeA, Type *InputTypeB, Type *AccumType,
       ElementCount VF, PartialReductionExtendKind OpAExtend,
       PartialReductionExtendKind OpBExtend, std::optional<unsigned> BinOp,
-      TTI::TargetCostKind CostKind) const;
+      TTI::TargetCostKind CostKind, std::optional<FastMathFlags> FMF) const;
```

Comment:
could you documentt the new argument and maybe also update the bit above that refers to only `llvm.vector.partial.reduce.add`